tests/failpoints_cases/test_split_region.rs

```diff
@@ -27,8 +27,9 @@ use test_raftstore::*;
 fn test_follower_slow_split() {
     let _guard = ::setup();
     let mut cluster = new_node_cluster(0, 3);
-    cluster.run();
     let pd_client = Arc::clone(&cluster.pd_client);
+    pd_client.disable_default_operator();
```

Comment:
Why is it still necessary?

Reply:
Because 3 peers will cause the test case fail because of unexpected vote-message count.